app/assets/javascripts/dashboard.js

```diff
@@ -156,10 +168,35 @@ MinionPoller = {
     masterHtml = '<input name="roles[master][]" id="roles_master_' + minion.id +
       '" value="' + minion.id + '" type="radio" disabled="" ' + checked + '>';
 
+    statusText = ''
```

Comment:
Status text will be empty if no update? Isn't it a bit weird to have an empty "Status" column? I would guess since it's being used to show the text "update available", we could use for other states too.